hasjob/templates/detail.html

```diff
@@ -231,7 +231,7 @@
             {%- endif %}
             {%- if is_siteadmin or post.admin_is(g.user) or (g.user and g.user.flags.is_employer_month) %}
               <div class="list-group-item no-border">
-                <i class="fa fa-fw fa-bar-chart-o"></i>&nbsp;&nbsp; <span title="Viewed › Opened form › Applied">{{ post.viewcounts['viewed'] }} › {{ post.viewcounts['opened'] }} › {{ post.viewcounts['applied'] }}</span> &middot;
+                <i class="fa fa-fw fa-bar-chart-o"></i>&nbsp;&nbsp; <span title="Viewed › Opened form › Applied">{{ get_post_viewcounts(post)['viewed'] }} › {{ get_post_viewcounts(post)['opened'] }} › {{ get_post_viewcounts(post)['applied'] }}</span> &middot;
```

Comment:
Fixed. Now calls `get_post_viewcounts` in the view and passes it on to the template.